copyright

```diff
@@ -1212,6 +1053,146 @@ Files:
 Copyright: Becca Tommaso (tommasobecca03@gmail.com)
 License: CC-BY-SA-4.0
 
+Files:
+ images/effect/remnant?afterburner/remnant?afterburner*
+ images/effect/mhd?spark*
+ images/land/nasa9*
+ images/hardpoint/annihilator?turret*
+ images/hardpoint/hai?ionic?turret*
+ images/hardpoint/inhibitor?turret*
+ images/hardpoint/ion?hail?turret*
+ images/hardpoint/ravager?turret*
+ images/outfit/inhibitor?turret*
+ images/outfit/ion?hail?turret*
+ images/hardpoint/shooting?star?flare/ss-rays*
+ images/outfit/overcharged?shield?module*
+ images/outfit/overclocked?repair?module*
+ images/outfit/ramscoop*
+ images/outfit/remnant?afterburner*
+ images/outfit/remnant?capital?license*
+ images/outfit/research?laboratory*
+ images/outfit/salvage?scanner*
+ images/outfit/tiny?remnant?engine*
+ images/outfit/void?rifle*
+ images/outfit/fragmentation?grenades*
+ images/outfit/nerve?gas*
+ images/outfit/catalytic?ramscoop*
+ images/outfit/anti-missile*
+ images/outfit/blaster?turret*
+ images/outfit/blaster*
+ images/outfit/breeder*
+ images/outfit/bunk?room*
+ images/outfit/dwarf?core*
+ images/outfit/electron?beam*
+ images/outfit/electron?turret*
+ images/outfit/fission*
+ images/outfit/flamethrower*
+ images/outfit/hai?ionic?blaster*
+ images/outfit/hai?ionic?turret*
+ images/outfit/heavy?anti-missile*
+ images/outfit/heavy?laser?turret*
+ images/outfit/heavy?laser*
```

Comment:
Should the "less specific file patterns before more specific file patterns" be applied within file stanzas?
For example, should these be reversed?
```suggestion
 images/outfit/heavy?laser*
 images/outfit/heavy?laser?turret*
```